package.json

```diff
@@ -61,7 +62,9 @@
     "ldapjs": "1.0.0"
   },
   "devDependencies": {
+    "autoprefixer": "6.4.1",
     "chai": "3.5.0",
+    "cssnano": "3.7.5",
```

Comment:
We minify most of our JS, and adding webpack would minify `lounge.js` too. Might as well do the same for CSS. `cssnano` does a bunch of optimizations on top of just removing whitespace.